app/v2/models/office_model.py

```diff
@@ -34,6 +34,11 @@ def from_json(self, json):
         self.id = json['id']
         return self
 
+    def edit(self, new_name):
```

Comment:
Consider renaming this method to something different, reason being, you're not overriding the base edit method. we only override when changing the logic which is not the case here. This is also the reason codacy is complaining. 
```suggestion
    def edit_office(self, new_name):
```